bookie_ml.py

```diff
@@ -0,0 +1,372 @@
+import numpy as np
+
+import nltk
+nltk.download('stopwords')
+from nltk.corpus import stopwords
+stop_words = set(stopwords.words('english'))
+nltk.download('averaged_perceptron_tagger')
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
+nltk.download('punkt')
+nltk.download('wordnet')
+from nltk.corpus import wordnet as wn
```

Comment:
Hmm, is there a better way to do this? Normally I would not mix code with import statements like this. But it looks like some imports are required to be downloaded first.